src/test/java/harness/TestUser.java

```diff
@@ -84,7 +86,8 @@ public void login() throws IOException {
     // get domain-wide delegated credentials for this user. use the same scopes that are requested
     // of CLI users when they login.
     GoogleCredentials googleCredentials = getCredentials(User.USER_SCOPES);
-    writeTerraOAuthCredentialFile(googleCredentials);
+    IdToken idToken = getIdToken(googleCredentials);
```

Comment:
opt Delete line 89 and just have `idToken = getIdToken(googleCredentials)` inside `writeTerraOAuthCredentialFile`